Dfe.Academies.Academisation.Service/Commands/Legacy/Project/LegacyProjectDeleteNoteCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Dfe.Academies.Academisation.Core;
+using Dfe.Academies.Academisation.Data;
+using Dfe.Academies.Academisation.Data.ProjectAggregate;
+using Dfe.Academies.Academisation.Domain.Core.ProjectAggregate;
+using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
+using Dfe.Academies.Academisation.IService.Commands.Legacy.Project;
+using Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ProjectAggregate;
+using Microsoft.EntityFrameworkCore;
+
+namespace Dfe.Academies.Academisation.Service.Commands.Legacy.Project
```

Comment:
I think the original intention was to have code in the service project call into the data project, which could use the AcademisationContext directly. Though it is a bit overcomplicated. This API needs a refactor to make it easier to use and consistent.

Reply:
Refactored to keep the AcademisationContext in the Data project.